eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -11,20 +11,23 @@ steps:
   # Installation steps need to be uncommented when switching to a newer SDK that's not available on DevOps agents
   - task: UseDotNet@2
     displayName: 'Use .NET Core SDK'
```

Comment:
Nit: (Maybe in a later update), change the display name to remove 'Core' from this and 5.0 below.